autocomplete/definitions/namedTypes/mwseMCMTemplate/new.lua

```diff
@@ -7,6 +7,14 @@ return {
 		tableParams = {
 			{ name = "name", type = "string", optional = true, description = "The name field is the mod name, used to register the MCM, and is displayed in the mod list on the lefthand pane." },
 			{ name = "label", type = "string", optional = true, description = "Used in place of `name` if that argument isn't passed. You need to pass at least one of the `name` and `label` arguments. If `headerImagePath` is not passed, a UI element will be created with `label` as text." },
+			{ name = "config", type = "table", optional = true, 
+				description = "Stores a config that should be used by this mods `Setting`s. Sub-configs can be accessed by passing a `configKey` to any `Page`s nested inside this template. \z
+					If provided, this config will be used to generate [`mwseMCMTableVariable`s](./mwseMCMTableVariable.md) for the  any [`mwseMCMSetting`s](./mwseMCMSetting.md) made inside this template."
+			},
+			{ name = "defaultConfig", type = "table", optional = true, 
```

Comment:
Looks like the `config` and `defaultConfig` fields weren't added to the mwseMCMTemplate definition.